stateMutators.js

```diff
@@ -285,35 +285,45 @@ export const calcLives = (state) => {
 
 export const calcLevel = (state) => {
     let level = state.gameLevel
+    let nextLevelScore = 10
     let levelTarget = state.levelTarget
     const score = state.score
 
     if (score >= 1 && score < 20) {
         level = 2
+        nextLevelScore = 20
     }
     if (score >= 3 && score < 35) {
         level = 3
+        nextLevelScore = 35
     }
     if (score >= 35 && score < 55) {
         level = 4
+        nextLevelScore = 55
     }
     if (score >= 55 && score < 80) {
         level = 5
+        nextLevelScore = 80
     }
     if (score >= 70 && score < 100) {
         level = 6
+        nextLevelScore = 100
     }
     if (score >= 100 && score < 135) {
         level = 7
+        nextLevelScore = 135
     }
     if (score >= 135 && score < 175) {
         level = 8
+        nextLevelScore = 175
     }
     if (score >= 175 && score < 220) {
         level = 9
+        nextLevelScore = 220
     }
     if (score >= 220) {
         level = 10
+        nextLevelScore = '∞'
```

Comment:
I don't really like this, as it makes `nextLevelScore` a `number` in most cases apart from this edge case, but I get why you're doing it. 

Let's keep it, but remember it in case we ever have to do some calculation with `nextLevelScore`

Reply:
Yea I feel the inconsistency too, to be honest, I was lazy so I just go with it 😅 , maybe we can find a better way later